core/math/math_funcs.cpp

```diff
@@ -138,14 +153,41 @@ double Math::ease(double p_x, double p_c) {
 		}
 	} else
 		return 0; // no ease (raw)
+}
+
+float Math::ease(float p_x, float p_c) {
```

Comment:
having double/float versions of this is kind of pointless, would just leave it as double